package net.tassia.math.error

open class VectorCrossProductException(

	message: String? = null, cause: Throwable? = null,

) : MathException(message, cause)
